### ISSUE TITLE
Dataset Urdu 

### ISSUE THREAD

**Issue by tayyabvohra** [keithito/tacotron#323]
I am trying to train the model on the Urdu Data set which is of 2 hours on 400K Steps . I have trained with `cleaners=transliteration_cleaners` I do have the results and the loss is about 0.06 but the problem is that the sentences are not been able to synthesized even it is not been able to pronounce.  What can be the possible ways to improve the model for language of Urdu

**Comment by aminbaig** [comment]
Hi Tayyaba,

First change your cleaners to "basic_cleaners", secondly, have you updated the symbols with unique characters from the urdu text?

Last but most important...your training data set is too small. with a decent dataset you should be able to attain alignment around 20k to 40k steps. 

Also how have you treated the audio file, noise, quality etc?

Please let me know if you are still working on this.
